### PR TITLE
fix(alert): add public --calcite-alert-offset-size css token

### PR DESCRIPTION
**Pull request by aPreciado88**
**Related Issue:** [#10520](https://github.com/Esri/calcite-design-system/issues/10520)

## Summary
Rename internal `--calcite-internal-alert-edge-distance` css token to `--calcite-alert-offset-size` and make it public.

### CODE REVIEW

**Comment by jcfranco** [comment]
@aPreciado88 Can you revisit the PR title to mention the prop that was added instead of focusing on exposing an internal prop?

**Review by alisonailea — APPROVED** [review]
🚀

File: packages/calcite-components/src/components/alert/alert.scss [hunk 1]
```diff
@@ -7,12 +7,12 @@
  * @prop --calcite-alert-background-color: Specifies the component's background color.
  * @prop --calcite-alert-corner-radius: Specifies the component's corner radius.
  * @prop --calcite-alert-shadow: Specifies the component's shadow.
+ * @prop --calcite-alert-offset-size: Specifies the component's bottom spacing
```
[the remark this answers is not in the document]
Comment (aPreciado88) [reply packages/calcite-components/src/components/alert/alert.scss:10]: @macandcheese I can update this! What would be a more appropriate description?

[the remark this answers is not in the document]
Comment (macandcheese) [reply packages/calcite-components/src/components/alert/alert.scss:10]: cc @geospatialem / @DitwanP but something like:

"Specifies the component's `"placement"` offset" or something?

**Review by macandcheese — APPROVED** [review]

**Comment by aPreciado88** [comment]
@ashetland @SkyeSeitz Could you please confirm that these are the usual chromatic false positives?

**Pull request merged**